.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
VSCode IDE setting file is a personal flavor, and better not be committed.

Reply:
I agree, thanks!